### PR TITLE
fix: cleanup cross-origin logs between tests

### PR DESCRIPTION
**Pull request by emilyrohrbough**
Clean-up cross-origin logs between tests.

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/app/src/runner/event-manager.ts [hunk 1]
```diff
@@ -631,6 +631,8 @@ export class EventManager {
         })
       }
 
+      // remove cross-origin log references from previous tests
```
Comment (emilyrohrbough) [inline packages/app/src/runner/event-manager.ts]:
```suggestion
      // remove cross-origin log references from previous test
```

File: cli/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
 ## 13.6.1
 
+_Released 12/19/2023 (PENDING)_
+
+**Bugfixes:**
+
+- Release cy.origin logs between tests. Fixed in [#28465](https://github.com/cypress-io/cypress/issues/28465).
```
Comment (emilyrohrbough) [inline cli/CHANGELOG.md]:
```suggestion
- Release cy.origin logs between tests. Fixed in [#28465](https://github.com/cypress-io/cypress/pull/28465).
```

File: cli/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,14 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
 ## 13.6.1
 
+_Released 12/19/2023 (PENDING)_
+
+**Bugfixes:**
+
+- Release cy.origin logs between tests. Fixed in [#28465](https://github.com/cypress-io/cypress/pull/28465).
```
Comment (jennifer-shehane) [inline cli/CHANGELOG.md]: @emilyrohrbough This isn’t super clear on what a user would see as the change to their tests from this description.

Comment (emilyrohrbough) [reply cli/CHANGELOG.md]: better? https://github.com/cypress-io/cypress/pull/28465/commits/1ef9c4abf5625370b7eee8a588d63f5ca2098c30#diff-59059a9fb84116ac978d9f97f3138d4741854d92c1b44c2db71f7e64c23e3a01R8

File: cli/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,14 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
 ## 13.6.1
 
+_Released 12/19/2023 (PENDING)_
+
+**Bugfixes:**
+
+- Clean up the cy.origin() logs references between tests to reduce memory consumption. Fixed in [#28465](https://github.com/cypress-io/cypress/issues/28465).
```
Comment (emilyrohrbough) [inline cli/CHANGELOG.md]:
```suggestion
- Clean up the cy.origin() logs references between tests to reduce memory consumption. Fixed in [#28465](https://github.com/cypress-io/cypress/pull/28465).
```

**Review by mschile — APPROVED** [review]

**Pull request closed without merging**